.storybook/package.json

```diff
@@ -40,7 +40,6 @@
 		"@etchteam/storybook-addon-status": "^5.0.0",
 		"@storybook/addon-a11y": "^8.4.6",
 		"@storybook/addon-actions": "^8.4.6",
-		"@storybook/addon-console": "^3.0.0",
```

Comment:
This plugin adds little value and is pretty slow as well so I included here to ensure reasonable load times for pages.